hoodie-common/src/main/java/com/uber/hoodie/common/table/HoodieTableMetaClient.java

```diff
@@ -79,13 +86,23 @@ public HoodieTableMetaClient(Configuration conf, String basePath)
   }
 
   public HoodieTableMetaClient(Configuration conf, String basePath,
-      boolean loadActiveTimelineOnLoad)
+      boolean loadActiveTimelineOnLoad) {
+    this(conf, basePath, loadActiveTimelineOnLoad, false, 0, 0, 0);
+  }
+
+  public HoodieTableMetaClient(Configuration conf, String basePath,
```

Comment:
This is polluting the constructor, can we wrap all consistency check configs into a single config and wire them here ?

Reply:
Fixed